.github/workflows/deployments.yml

```diff
@@ -540,8 +567,13 @@ jobs:
             refs="$refs $registry@$digest"
           done
 
-          branch="${{ github.ref_name }}"
-          stitched_tag="$registry:cu${cuda_major}-${branch}-base"
+          if [[ "${{ steps.metadata.outputs.is_versioned }}" == "true" ]]; then
+            ref_name="${{ steps.metadata.outputs.image_tag }}"
+          else
+            ref_name=`echo ${{ github.ref_name }} | tr '/' '-'`
+          fi
+
+          stitched_tag="$registry:cu${cuda_major}-${ref_name}-base"
```

Comment:
You've got a duplication of the cu*- part here, same above

Reply:
good catch, thanks.